pkgs/applications/misc/parsec/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, lib
+, buildFHSUserEnv
+, parsecDrv ? null
```

Comment:
What is this?

Reply:
> I don't see much value on this, considering that we already have things like `overrideAttrs`.

The problem here is that it has three stages

With overrideAttrs with two stages you can override both stages but with three you can't override the first one

Reply:
You still can, just override the `installPhase`.

Reply:
Then this needs an explanation because right now the average person has absolutely no idea what is happening here.